build-logic/performance-testing/src/main/kotlin/gradlebuild/performance/tasks/DetermineBaselines.kt

```diff
@@ -39,6 +38,12 @@ abstract class DetermineBaselines @Inject constructor(@get:Internal val distribu
     @get:Internal
     abstract val determinedBaselines: Property<String>
 
+    @get:Internal
```

Comment:
I suppose these two properties should be `@Input`s, right?

Reply:
Why? this task doesn't declare output and enable caching.

Reply:
Why not? If at some point it does, then you don't need to think about what are inputs and which are not. From a practical perspective, I think it is right that it doesn't matter. Though from a documentation perspective I think it does: which properties are you supposed to set on the task and which one are the one which you shouldn't.

I don't feel very strongly that you should change this, so it is up to you.